app/Config/Exceptions.php

```diff
@@ -60,7 +60,7 @@ class Exceptions extends BaseConfig
      * Use this option to temporarily cease the warnings and instead log those.
      * This option also works for user deprecations.
      */
-    public bool $logDeprecationsOnly = false;
+    public bool $logDeprecationsOnly = true;
```

Comment:
I realize the name predates this PR but the phrasing seems to imply "only log things that are deprecations". How about simply "logDeprecations" or even "convertDeprecations"?